crates/bitwarden/src/vault/send.rs

```diff
@@ -135,6 +135,15 @@ impl Send {
         let key = derive_shareable_key(key.try_into().unwrap(), "send", Some("send"));
         Ok(key)
     }
```

Comment:
Can this be made private now? I think all calls to `Send::get_key` can safely be replaced by `get_encryption`.

Reply:
Good point, changed now